Mage.Sets/src/mage/cards/b/BillFernyBreeSwindler.java

```diff
@@ -26,7 +27,6 @@
  * @author bobby-mccann
  */
 public final class BillFernyBreeSwindler extends CardImpl {
-    private static final FilterControlledPermanent horseYouControl = new FilterControlledPermanent(SubType.HORSE);
```

Comment:
Prefer to keep filters static to avoid unnecessary copying of filter objects on each game state iteration